synapse/models/economic.py

```diff
@@ -61,10 +67,29 @@ def getModelDefs(self):
                 ('econ:fin:tick', ('guid', {}), {
                     'doc': 'A sample of the price of a security at a single moment in time'}),
 
-                # econ:acct:bill
-                # econ:bank:us:aba:rtn ( ABA Routing Number )
-                # econ:bank:us:account = (econ:bank:us:aba:rtn, acct)
-                # econ:bank:swift:...
+                ('econ:bank:account:type:taxonomy', ('taxonomy', {}), {
```

Comment:
Does econ:acct:payment also need a :type to capture the type of transaction (e.g.,  debit)?

Reply:
Out of scope for this PR. If you believe it should, feel free to submit a ticket. :+1: